app/views/obituaries/email/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+  <div class="col-sm-9 col-md-7 col-lg-8 mx-auto">
+    <br><h1 class="card-title text-center">Share this Obituary by Email</h1>
+    <div class="card card-search my-5">
+      <%= form_tag "/obituaries/#{@obit.id}/share", method: :post, class: 'submit-form' %>
+        <div class="card-body" style= "background-color:white; text-align:center;">
+          <p class="card-title">You are sharing the following obituary in an email:</p>
+          <a href="/obituaries/<%=@obit.id%>" class="card-link"><%=@obit.first_name%> <%=@obit.last_name%>         "<%=@obit.description.first(60)%> ..." </a>
+
+          <div class="form-group-search row">
+            <div class="col col-form-label-lg">
+              <%= label_tag 'Recipient\'s Name' %>
+              <%= text_field_tag :recipient_name, nil, placeholder: 'Abraham Lincoln', class: "form-control" %>
```

Comment:
Thank you @stellakunzang  for the name suggestiong.

Reply:
![](https://media2.giphy.com/media/QWwEdgDbYjFbfOMJ3z/giphy.gif?cid=5a38a5a2rga94co2t79tnrtjbfylohysir2z8x23xsclyo9f&rid=giphy.gif)